lib/ast-utils.js

```diff
@@ -24,6 +24,8 @@ const bindOrCallOrApplyPattern = /^(?:bind|call|apply)$/;
 const breakableTypePattern = /^(?:(?:Do)?While|For(?:In|Of)?|Switch)Statement$/;
 const thisTagPattern = /^[\s*]*@this/m;
 
+
+const COMMENTS_IGNORE_PATTERN = "^\\s*(?:eslint|jshint\\s+|jslint\\s+|istanbul\\s+|globals?\\s+|exported\\s+|jscs|falls?\\s?through)";
```

Comment:
I don't think the falls through comment should be included in this. It's possible for someone to not have the falls-through rule enabled and to have a comment that begins with "falls through". It would be unexpected if that comment wasn't flagged by this rule.